### PR TITLE
Updating all locations where DIRECTION was used instead of DECORATION

### PR DESCRIPTION
**Pull request by dzaro-dev**
As per @MrD-RC 's feedback in [PR#10031](https://github.com/iNavFlight/inav/pull/10031), I've updated all the locations where either

`SYM_AH_DIRECTION_UP` or `SYM_AH_DIRECTION_DOWN` were used to:

`SYM_AH_DECORATION_UP` and `SYM_AH_DECORATION_UP`.

I closed all the other PR's because this should be all under one PR and not separate ones.

This matches what is written in the the [INAV Character Map.md](https://github.com/iNavFlight/inav-configurator/blob/b717b2b394fd0f9e6c66927342ad2fc3c93a40f7/resources/osd/INAV%20Character%20Map.md?plain=1) and ensures that what is written there is also in the firmware code.

This also allowed the enabling of the direction scrolling arrows in `displayport_msp_bf_compat.c` when using DJI O3, as symbols are available to replace those blanks.

Hope this helps.

### CODE REVIEW

File: src/main/io/displayport_msp_bf_compat.c [hunk 1]
```diff
@@ -90,13 +90,13 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
...
+*/
     case SYM_AH_DECORATION_UP:
-        return BF_SYM_AH_DECORATION_UP;
+        return BF_SYM_ARROW_SMALL_UP;
 
     case SYM_AH_DECORATION_DOWN:
-        return BF_SYM_AH_DECORATION_DOWN;
-*/
+        return BF_SYM_ARROW_SMALL_DOWN;
+
```
Comment (MrD-RC) [inline src/main/io/displayport_msp_bf_compat.c]: This should be a separate PR. As it is not in the original scope.

**Review by MrD-RC — CHANGES_REQUESTED** [review]

File: src/main/io/displayport_msp_bf_compat.c (hunk 1 above)
Comment (dzaro-dev) [reply src/main/io/displayport_msp_bf_compat.c]: @MrD-RC No problem, I'll make a separate PR for this one. 

Is ti possible to remove these changes from this PR? 

Sorry, I'm not very well versed in GitHub things.

Comment (rmaia3d) [reply src/main/io/displayport_msp_bf_compat.c]: Just revert these highlighted changes in the file, save and create/push a new commit.

Comment (dzaro-dev) [reply src/main/io/displayport_msp_bf_compat.c]: Got it, thank you, will do now. 👍

**Comment by bishko** [comment]
@MrD-RC and @rmaia3d Is this ok now?
